src/addon/cc-header-addon.js

```diff
@@ -182,12 +181,10 @@ export class CcHeaderAddon extends LitElement {
           --cc-gap: 0.5rem;
           --cc-align-items: center;
           align-items: center;
-          background-color: var(--color-bg-neutral);
-          box-shadow: inset 0 6px 6px -6px #a4b1c9;
+          background-color: var(--cc-color-bg-neutral);
+          box-shadow: inset 0 6px 6px -6px rgba(0, 0, 0, 0.4);
           box-sizing: border-box;
-          color: #2e2e2e;
```

Comment:
this color is almost the same as our default text color which is inherited from host anyway.
The color is managed by `<cc-zone>` anyway so it would not make a difference to set it here.

Reply:
It is related to this area:
![image](https://user-images.githubusercontent.com/100240294/177791143-c8d5cd1f-0dc4-4f76-8704-b627b0341ef0.png)